Lesson2/src/main/java/pl/slvd/university/Faculties.java

```diff
@@ -0,0 +1,55 @@
+package pl.slvd.university;
+
+import java.util.Arrays;
+
+import java.util.Locale;
+import java.util.Scanner;
+
+public class Faculties {
+
+    public enum Faculty {
+        INSTRUMENT, VOCAL, THEATRE, CHOREOGRAPHY;
+
```

Comment:
add some parameters to enums, so to make them different from each other not only by the names.

Reply:
I wanted to contact the Dean's office (Deanery), each faculty will have its own Dean. Or is this not enough?

Reply:
Yes, the Faculty class is here for a simple purpose and will probably take on some sort of full role if it expands the project with Students, Professors, Deanery, Corps, etc.